cloudbuild.yaml

```diff
@@ -0,0 +1,28 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: [
+    'build',
+    '-t', 'us-central1-docker.pkg.dev/$PROJECT_ID/topicctl/image:$COMMIT_SHA',
+    '-t', 'us-central1-docker.pkg.dev/$PROJECT_ID/topicctl/image:latest',
```

Comment:
do you actually want this `latest` tag?

Reply:
I think it is not a bad idea. It tells us which image is the latest one from main as the last generated image can come from a PR branch. Also useful to run the latest version locally.